src/extension.ts

```diff
@@ -2,6 +2,7 @@
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
+import { XLog } from './tool';
```

Comment:
Are you going to develop new logging framework? Why not just use existing one? For now please remove tool.ts file and all references to it.

Reply:
I will remove tool.ts file.